crates/sui/tests/protocol_version_tests.rs

```diff
@@ -593,6 +595,34 @@ mod sim_only_tests {
         assert!(matches!(system_state, SuiSystemState::SimTestV2(_)));
     }
 
+    #[sim_test]
+    async fn check_framework_snapshot_compatibility() {
+        let snapshot_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+            .join("../sui-framework-snapshot/bytecode_snapshot");
```

Comment:
nit: `extend` to avoid hard coded path separators?
```suggestion
            .extend(["..", "sui-framework-snapshot", "bytecode_snapshot"]);
```